libbeat/common/dtfmt/elems.go

```diff
@@ -194,6 +187,7 @@ func (n paddedNumber) compile() (prog, error) {
 	if n.div == 0 {
 		return makeProg(opNumPadded, byte(n.ft), byte(n.maxDigits))
 	}
+	fmt.Println("paddedNumber compile div: ", n.div, byte(n.div))
```

Comment:
This code is my problem. This div variable is large, so int convert to byte error. The nanoseconds use this, so i don't know how to deal with. Thanks.